src/components/profile/business-profile.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+// import {useSelector} from "react-redux";
+import {Link} from "react-router-dom";
+import restaurants from "../data/restaurants.json";
+import critics from "../data/critic-users.json";
+import ReviewList from "../reviews/reivew-list";
+import BusinessInfo from "../restaurant/business-info";
+import BusinessPost from "../restaurant/business-post";
+import FeaturedItems from "../restaurant/featured-items";
+import "../restaurant/business-home.css";
+
+// hardcode restaurant input, need to change later
+const BusinessProfile = ({restaurant=restaurants[0]}) => {
+    // const restaurant = useSelector(state => state.restaurant);
+
+    return (
+    <div className="border ttr-border-radius">
+        <div className="position-relative ttr-banner d-flex justify-content-center">
+            <img src={`/images/${restaurant.bannerPicture}`}
+                 className="ttr-border-radius ttr-banner-width mt-3" height={200}/>
+            <img className="ttr-portrait position-absolute start-0 ms-5"
+                 src={`/images/${restaurant.profilePicture}`}/>
+        </div>
+        <div className="m-3 position-relative">
+                <span className="h5 fw-bolder">{restaurant.name} {restaurant.handle}
+                </span><br/>
+            {/*Profile edit button*/}
+            <Link to="./edit-restaurant">
```

Comment:
I think Yena implemented the path to edit business profile as /restaurant/:rid/edit. Let's agree on a path before merging

Reply:
Sounds good.

Reply:
I made the revision on my path according to the conversation in request#6.